sdks/java/io/iceberg/src/main/java/org/apache/beam/sdk/io/iceberg/IcebergWriteSchemaTransformProvider.java

```diff
@@ -57,8 +66,59 @@ public String description() {
         + "{\"table\" (str), \"operation\" (str), \"summary\" (map[str, str]), \"manifestListLocation\" (str)}";
   }
 
+  @DefaultSchema(AutoValueSchema.class)
+  @AutoValue
+  public abstract static class Configuration {
+    public static Builder builder() {
+      return new AutoValue_IcebergWriteSchemaTransformProvider_Configuration.Builder();
+    }
+
+    @SchemaFieldDescription("Identifier of the Iceberg table.")
+    public abstract String getTable();
+
+    @SchemaFieldDescription("Name of the catalog containing the table.")
+    @Nullable
```

Comment:
Make the annotation modify the type, not the method, like so:

`public abstract @Nullable String`

(you will need to switch to the checkerframework annotation for this)

Reply:
Thanks for this. AutoValue wasn't recognizing checkerframework's `Nullable` annotation as it is placed here, so I switched to javax. Looks like I just had the annotation in the wrong place.